src/scss/2022/objects/_grid.scss

```diff
@@ -18,33 +29,50 @@
   row-gap: var(--gap-row, var(--gap));
   column-gap: var(--gap-col, var(--gap));
   
-  @media (min-width: 50rem) {
+  @media (min-width: mq('sm')) {
+    --grid-size: 4;
+    row-gap: var(--gap-row-sm, var(--gap-sm));
+    column-gap: var(--gap-col-sm, var(--gap-sm));
+  }
+
+  @media (min-width: mq('md')) {
     --grid-size: 8;
     row-gap: var(--gap-row-md, var(--gap-md));
     column-gap: var(--gap-col-md, var(--gap-md));
   }
   
-  @media (min-width: 70rem) {
+  @media (min-width: mq('lg')) {
     --grid-size: 12;
     row-gap: var(--gap-row-lg, var(--gap-lg));
     column-gap: var(--gap-col-lg, var(--gap-lg));
   }
 
-
-
   &__col {
+    // Default
     --start: 1;
     --span: 4;
     --align: 'top';
     --row-span: 1;
     --row-start: auto;
     --order: auto;
-    --start-md: var(--start);
-    --span-md: var(--span);
-    --align-md: var(--align);
-    --row-span-md: var(--row-span);
-    --row-start-md: var(--row-start);
-    --order-md: var(--order);
+
+    // Default @SM
+    --start-sm: var(--start);
+    --span-sm: var(--span);
+    --align-sm: var(--align);
+    --row-span-sm: var(--row-span);
+    --row-start-sm: var(--row-start);
+    --order-sm: var(--order);
+
+    // Default @MD
+    --start-md: var(--start-sm);
+    --span-md: var(--span-sm);
+    --align-md: var(--align-sm);
+    --row-span-md: var(--row-span-sm);
+    --row-start-md: var(--row-start-sm);
+    --order-md: var(--order-sm);
```

Comment:
Just checking: it looks to me like all of these values would end up being the same at all breakpoints since they're all chained from the default values. Is that right?

Reply:
Yeah! That’s the idea. That provides the base and then allows me to change it at will and then larger sizes will inherit whatever changes are made at smaller sizes.